starlite/kwargs.py

```diff
@@ -171,10 +176,18 @@ async def resolve_dependency(
         dependency: An instance of [Dependency][starlite.kwargs.Dependency]
         connection: An instance of [Request][starlite.connection.Request] or [WebSocket][starlite.connection.WebSocket].
         kwargs: Any kwargs to pass to the dependency, the result will be stored here as well.
+        cleanup_group: DependencyCleanupGroup to which generators returned by `dependency` will be added
     """
     signature_model = get_signature_model(dependency.provide)
     dependency_kwargs = signature_model.parse_values_from_connection_kwargs(connection=connection, **kwargs)
-    kwargs[dependency.key] = await dependency.provide(**dependency_kwargs)
+    value = await dependency.provide(**dependency_kwargs)
+    if isgenerator(value):
```

Comment:
```suggestion

    if isgenerator(value):
```

Reply:
Is there something wrong on my end? Those lines are the same for me.

Reply:
@provinzkraut I think it is about visual highlight for the `if` with an empty line before it.

Reply:
Oh. Yes. Thanks. I'm blind :upside_down_face: